code/modules/food_and_drinks/recipes/tablecraft/recipes_guide.dm

```diff
@@ -515,7 +519,7 @@
 	result = /datum/reagent/consumable/flour
 
 /datum/crafting_recipe/food/grinder/butter
-	reqs = list(/datum/reagent/consumable/milk = 15)
+	reqs = list(/datum/reagent/consumable/milk = 25)
```

Comment:
hey that's weird...

Reply:
That shows up as the proper 25 on my machine? the hell

Reply:
Oh wait, i think it's another filepath